### ISSUE TITLE
The first Codelab example at Step 6 is hard to follow

### ISSUE THREAD

**Issue by jeejeestudio** [flutter/website#6836]
Hi, please revise step 6 of this tutorial https://codelabs.developers.google.com/codelabs/first-flutter-app-pt1#5
. It is hard to follow for a newcomer to flutter and unclear where to exactly add the lines. Yes you can copy the script from Github, but that's not learning. Also the script looks not exactly the same, so that confuses even more. It works yes, but what did I learn?

**Comment by danagbemava-nc** [comment]
Hi @jeejeestudio, thanks for the feedback. 

Can you provide a bit more detail as to what made it difficult for you to follow?

Thank you

**Comment by jeejeestudio** [comment]
Hi Daniel,
    
      
    thanks for replying, I
        really appreciate that.
    
      At Part1 Step 6
          Create an infinite scrolling ListView:
    
    Add the entire _buildSuggestions function.
      
      In the _RandomWordsState class, add the following function,
        deleting the comments, if you prefer:
    It was for me unclear where exactly to put it.
    
      next also Part 1 Step 6::
    
    Add a _buildRow function to _RandomWordsState:
    
    this is also unclear where to put it, between or after which
      brackets? Also in the example file it's not even there.
    
      So at this point one has downloaded the main.dart because
        following the tutorial  was not clear, you go to Part 2 step 4
        and you're blocked again:
    
    In the _buildRow function, add an alreadySaved check to ensure
        that a word pairing has not already been added to favorites.
    Where is this _buildRow ? it's not in the main.dart and if I need
      to add it, where do I add it?
    
    
    Perhaps things have changed due to new versions of Flutter, I
      don't know, but a newcomer can't follow.
    
    Thanks for your help and hopefully you can make it more clearer.
    
    
    Kind regards,
    Jerry
    
    Op 21-2-2022 om 14:37 schreef Daniel
      Agbemava:
    
    
      
      Hi @jeejeestudio,
        thanks for the feedback.
      Can you provide a bit more detail as to what made it
        difficult for you to follow?
      Thank you
      —
        Reply to this email directly, view it on GitHub, or unsubscribe.
        Triage notifications on the go with GitHub Mobile for iOS or Android.
        
        You are receiving this because you were mentioned.Message
          ID: ***@***.***>
      [
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/flutter/website/issues/6836#issuecomment-1046889908",
"url": "https://github.com/flutter/website/issues/6836#issuecomment-1046889908",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by danagbemava-nc** [comment]
Thanks for the feedback. 

/cc @domesticmouse

**Comment by parlough** [comment]
Thanks for the feedback! This codelab has been superseded by a new [Your first Flutter app](https://codelabs.developers.google.com/codelabs/flutter-codelab-firs) codelab which has been completely rewritten and seems to do a good job on explaining the changes to be made and why.

The goal of these initial codelabs is mostly to provide some exposure to Flutter rather than go super in-depth on the topics and I think this new one does so well. Let us know if you (or anyone else reading this) has any feedback on the new codelab though. Thanks again :D

**Comment by jeejeestudio** [comment]
Thank you so much for your great effort! I really appreciate this hard work. I have my first Flutter app on Google Play now, but I will check out these updated lessons as there is still so much to learn. Thanks again!

**Comment by parlough** [comment]
> Thank you so much for your great effort! I really appreciate this hard work. I have my first Flutter app on Google Play now, but I will check out these updated lessons as there is still so much to learn. Thanks again!

@jeejeestudio Congratulations on your first published app!
